### DISCUSSION TITLE
Module map file not found, can't build from xcode, can build with npm run ios

### DISCUSSION THREAD

**Discussion by sgrutman978** [rnmapbox/maps#3404] [Q&A]
Hi everyone. So if I do npm run ios, its builds perfectly fine on simulator and on my own physical device. However if I try to run it from xcode, I get this error (see image). I have tried removing the cache in the derived data folder, tried installing the package directly in the package manager, I have tried a number of different things but I cannot figure this out. Please help. Thank you

<img width="899" alt="image" src="https://github.com/rnmapbox/maps/assets/5942697/c91d911b-1bf6-4909-b3db-7210864009b8">

**Comment by mfazekas** [comment]
> PLEASE HELP: [#3404](https://github.com/rnmapbox/maps/discussions/3404)

@sgrutman978 pls don't spam to unrelated discussions.

**Reply by sgrutman978** [reply]
I didn't realize a single message in a single thread was spamming but... sorry. Can you help me out?
